bin/express

```diff
@@ -49,7 +49,7 @@ program
   .option('    --pug', 'add pug engine support', renamedOption('--pug', '--view=pug'))
   .option('    --hbs', 'add handlebars engine support', renamedOption('--hbs', '--view=hbs'))
   .option('-H, --hogan', 'add hogan.js engine support', renamedOption('--hogan', '--view=hogan'))
-  .option('-v, --view <engine>', 'add view <engine> support (ejs|hbs|hjs|jade|pug|twig|vash) (defaults to jade)')
+  .option('-v, --view <engine>', 'add view <engine> support (ejs|hbs|hjs|jade|pug|twig|vash|none) (defaults to jade)')
```

Comment:
Yea, idk, it's more about what is less confusing to the general Express users. The only thing I really have is that all the previous issues / PR called it "static" and no one until this one thought to call it "none", so that's all I have to go on.